source/standards/aws-account-management.html.md.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: AWS Account Management
+expires: 2018-10-18
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+GDS manage a number of Amazon Web Services (AWS) accounts.
+User accounts are managed centrally in a "base account" from which users
+can access the target accounts through a process called "assuming roles".
+
+## Advice on managing cross account access
+
+Cross account access allows user accounts in one AWS account (the *base* account) to access resources in another AWS account (the *target* account).
+
+Access to the resources within the target AWS account is controlled through IAM roles which are managed within that target AWS account.
+
+The IAM roles are used to establish trust relationships between the target (*trusting*) AWS account and the base (*trusted*) AWS account.
+
+It is the target AWS accounts responsibility to manage access to it's resources. Although user accounts are managed within the base AWS account access to resources is managed by the target AWS account in which those resources reside.
```

Comment:
I've pushed a commit that updates the introduction to explain why we're recommending teams use multiple AWS accounts, rather than multiple users and roles within one account.

The main reason is it enforces a strong, predictable security boundary between sensitive and non-sensitive accounts (e.g. production and test).